css/CalendarMonth.scss

```diff
@@ -36,6 +36,20 @@
   margin-bottom: 2px;
 }
 
+.CalendarMonth__weeknumber {
+  padding: 0;
+  box-sizing: border-box;
+  color: $react-dates-color-gray;
+  cursor: default;
+  width: 39px;
+  height: 38px;
+
+  user-select: none;
```

Comment:
Can you test that this `user-select` is still necessary given https://github.com/airbnb/react-dates/pull/74 and follow up commit?